CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## Unreleased
+## 21.48.0
 
 * Expand js classes code ([PR #1513](https://github.com/alphagov/govuk_publishing_components/pull/1513))
```

Comment:
I know it's minor, but can we add this fix 'Tweak size of action link subtext divider #1512' to the changelog too – it's a visual change affecting a component.

Reply:
Sure, added.